seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/oceanbase/OceanBaseMySqlCatalog.java

```diff
@@ -217,4 +218,49 @@ public String getExistDataSql(TablePath tablePath) {
                 "SELECT * FROM `%s`.`%s` LIMIT 1;",
                 tablePath.getDatabaseName(), tablePath.getTableName());
     }
+
+    public CatalogTable getCatalogTable(Connection defaultConnection, String sqlQuery)
```

Comment:
how about put this part into `public CatalogTable getTable(String sqlQuery) throws SQLException {`?

Reply:
ok